dev/benchmarks/macrobenchmarks/test/fullscreen_textfield_perf_e2e.dart

```diff
@@ -12,11 +12,7 @@ void main() {
   macroPerfTestE2E(
     'fullscreen_textfield_perf',
     kFullscreenTextRouteName,
-    // The driver version doesn't have this delay because the delay caused
-    // by the communication between the host and the test device is long enough
-    // for the driver test, but there isn't such delay in this host independent
-    // test.
-    pageDelay: const Duration(milliseconds: 50),
+    // pageDelay: const Duration(seconds: 1),
```

Comment:
Is this meant to be removed?

Reply:
No, actually, I think I was doing a quick manual A/B test before pushing it and forgot to put it back.

Reply:
Restored...